src/pages/index.tsx

```diff
@@ -1,38 +1,28 @@
-import { AxiosError } from 'axios';
 import { Head } from 'components/Head';
 import { DashboardTable } from 'components/pages/dashboard/DashbardTable';
 import { DashboardTopArea } from 'components/pages/dashboard/DashboardTopArea';
+import { useFetchPollData } from 'hooks/useFetchPollData';
 import { BaseLayout } from 'layouts/BaseLayout';
-import React, { useState } from 'react';
-import { poll } from 'services/pollService';
-import { PollResponse } from 'types/api';
+import React from 'react';
 
 function HomePage() {
-  const [data, setData] = useState<PollResponse>({
-    done: false,
-    proc_start: null,
-    proc_end: null,
-    plants: [],
-  });
+  const { pollData, isLoading, error } = useFetchPollData();
```

Comment:
As I mentioned in the google doc file, with each data fetch, I could compare all newly completed processes and provide the user with feedback via snackbars or other methods. I don't believe you replied to that part in the google doc. Could you confirm if you are interested in that feature or not?

Reply:
Nice to have, but definitely not a necessity. Low priority.